site/content/blog/2018-03-12-GA-GTM.adoc

```diff
@@ -0,0 +1,90 @@
+---
+author: Benjamin
+cover: ga-gtm-banner
+date: '2018-04-09'
+tags:
+- GA
+- GTM
+title: 'Google Analytics via Google Tag Manager'
+url: /2018/04/09/ga-gtm/
+---
+
+C'est en branchant https://www.facebook.com/business/a/facebook-pixel[PIXEL] sur un projet que j'ai enfin compris et vraiment utilisé *Google Tag Manager*.
```

Comment:
I think you could put the full name the first time "Facebook PIXEL" (I clicked to understand what it was)